modules/core/arrow-core/src/main/kotlin/arrow/core/extensions/sequenceK.kt

```diff
@@ -71,10 +76,15 @@ interface SequenceKEq<A> : Eq<SequenceK<A>> {
 
   fun EQ(): Eq<A>
 
-  override fun SequenceK<A>.eqv(b: SequenceK<A>): Boolean =
-    zip(b) { aa, bb -> EQ().run { aa.eqv(bb) } }.fold(true) { acc, bool ->
-      acc && bool
-    }
```

Comment:
This is kind of concerning, this should have failed tests, for different sized lists. Will check this later on...

Reply:
Welp our tests are flawed... Constructor functions like `cf: (Int) -> Kind<F, Int>` are heavily misused, similar to applicative instances. This needs to change because currently the guarantees that our tests provide are very very weak. I'll open a ticket for discussion sometime later today...